tools/test/audio/comp_run.sh

```diff
@@ -121,18 +123,42 @@ run_testbench ()
 
 parse_args "$@"
 
-# Paths
-HOST_ROOT=../../testbench/build_testbench
-HOST_EXE=$HOST_ROOT/install/bin/testbench
-HOST_LIB=$HOST_ROOT/sof_ep/install/lib
-TPLG_LIB=$HOST_ROOT/sof_parser/install/lib
+# Path to topologies
 TPLG_DIR=../../build_tools/test/topology
 
+# Testbench path and executable
+if [[ -z $XTRUN ]]; then
+    TESTBENCH=../../testbench/build_testbench/install/bin/testbench
+else
+    BUILD_DIR=../../testbench/build_xt_testbench
+    TESTBENCH="$BUILD_DIR"/testbench
+    source "$BUILD_DIR"/xtrun_env.sh
+    XTRUN_CMD=$XTENSA_PATH/$XTRUN
+    if $VALGRIND; then
+	>&2 printf "WARNING: Ignoring VALGRIND with xt-run\n"
+	VALGRIND=false
```

Comment:
indent

Reply:
Same, I have no idea what's wrong with this indent. My emacs bash mode indents this way.

Reply:
Mix of tabs and spaces again?